controllers/comment.js

```diff
@@ -0,0 +1,155 @@
+import { Comment, User } from '../models';
+
+/**
+ * @class CommentController
+ *  @override
+ * @export
+ *
+ */
+export default class CommentController {
+  /**
+   * @description - Creates an article comment
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   *
+   * @memberof CommentController
+   *
+   * @returns {object} Article comment
+   */
+  static async createComment(req, res) {
+    const { id } = req.user;
+    const { slug: articleSlug } = req.params;
+    const { comment } = req.body;
+    try {
+      const newComment = await Comment.create({
+        userId: id, articleSlug, comment
+      });
+      return res.status(205).json({
+        success: true,
+        message: 'New article comment created successfully',
+        comment: newComment,
+      });
+    } catch (error) {
+      return res.status(500).json({
+        success: false,
+        errors: [error.message]
+      });
+    }
+  }
+
+  /**
+   * @description - Gets an article's comments
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   *
+   * @memberof CommentController
+   *
+   * @returns {object} Article comment
+   */
+  static async getComments(req, res) {
+    const { slug: articleSlug } = req.params;
+    try {
+      const findComment = await Comment.findAll({
+        raw: true,
+        where: {
+          articleSlug
+        },
+        include: [
+          {
+            model: User,
+            attributes: ['username', 'bio', 'email']
+          }
+        ]
+      });
+      return res.status(205).json({
+        success: true,
+        message: 'Comments returned successfully',
+        comments: findComment,
+      });
+    } catch (error) {
+      return res.status(500).json({
+        success: false,
+        errors: [error.message]
```

Comment:
confirm that the error object has a message property.

Reply:
That seems to be the standard error format. See more here: https://github.com/sequelize/sequelize/blob/3e5b8772ef75169685fc96024366bca9958fee63/lib/errors.js#L49